web-bundle/src/main/java/com/graphhopper/resources/IsochroneResource.java

```diff
@@ -169,6 +172,7 @@ public Response doGet(
         }
         ObjectNode json = JsonNodeFactory.instance.objectNode();
 
+        sw.stop();
```

Comment:
Took was always zero, because the sw wasn't stopped before reading it on line 183.